src/render/image_manager.ts

```
@@ -88,7 +88,7 @@ class ImageManager extends Evented {
     }
 
     addImage(id: string, image: StyleImage) {
-        assert(!this.images[id]);
+        naiveAssert(!this.images[id]);
```

Same as above, throw instead of silently failing or failing with an cumbersome error message...

---

now throwing